<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_sign_in">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>498</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
        stop:0.0112994 rgba(64, 145, 252, 255), 
        stop:1 rgba(255, 255, 255, 255));
    color: rgb(255, 255, 255);
    border: 0px groove gray;
    border-radius: 7px;
    padding: 4px 6px;
    font: 14pt &quot;Candara&quot;;
    box-shadow: 3px 3px 6px rgba(0, 0, 0, 50);
}

/* 鼠标悬停效果 */
QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
        stop:0 rgba(72, 155, 255, 255), 
        stop:1 rgba(240, 240, 240, 255));
    color: rgb(240, 240, 240);
    border: 1px solid rgba(64, 145, 252, 200);
}

/* 按下效果 */
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
        stop:0 rgba(50, 130, 240, 255), 
        stop:1 rgba(220, 220, 220, 255));
    color: rgb(220, 220, 220);
    padding: 6px 8px; /* 模拟下沉效果 */
    border: 1px solid rgba(50, 130, 240, 200);
}
</string>
    </property>
    <property name="text">
     <string>登 录</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_txt">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 50pt &quot;千图笔锋手写体&quot;;</string>
    </property>
    <property name="text">
     <string>欢迎使用云程</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_usrname">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>388</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 10pt &quot;Candara&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请 输 入 你 的 邮 箱</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pwd">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>448</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 10pt &quot;Candara&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请 输 入 你 的 密 码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>-10</y>
      <width>541</width>
      <height>781</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_tips">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>邮箱密码登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign_up">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
    font-size: 12px;
    color: blue; /* 默认文字颜色 */
}

/* 鼠标悬停效果 */
QPushButton:hover {
    color: blue; /* 鼠标悬停时文字颜色变蓝 */
    text-decoration: underline; /* 鼠标悬停时添加下划线 */
}

/* 按下效果 */
QPushButton:pressed {
    color: darkblue; /* 按下时文字变为深蓝色 */
    padding: 2px; /* 模拟按钮按下的轻微位移 */
}
</string>
    </property>
    <property name="text">
     <string>去注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>559</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px</string>
    </property>
    <property name="text">
     <string>没有账号？</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
